src/Control/Monad/Error/Hoist.hs

```diff
@@ -41,6 +42,12 @@ instance MonadError e' m ⇒ HoistError m (Either e) e e' where
 instance (m ~ n, MonadError e' m) ⇒ HoistError m (EitherT e n) e e' where
   hoistError f = eitherT (throwError . f) return
 
+instance MonadError e' m ⇒ HoistError m (ExceptT e Identity) e e' where
```

Comment:
`type Except e = ExceptT e Identity`, so I made the instance analogous to `Either` for this case.